.env

```
@@ -12,9 +12,12 @@ DELETE_SINGLE_NOTIFICATION="notifications"
 MARK_NOTIFICATION_AS_READ_URL="notifications/mark-read"
 FORGET_PASSWORD_URL="auth/forgotPassword"
 RESET_PASSWORD_URL="auth/resetPassword"
+SPECIFIC-PRODUCT="products/673e1cd711599201718280fb"
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The SPECIFIC-PRODUCT key has incorrect delimiter

(IncorrectDelimiter)